tests/testthat/test-shinytest2-snapshot.R

```diff
@@ -0,0 +1,188 @@
+testthat::test_that("e2e: Create empty snapshot", {
```

Comment:
Please remove this file and let's come back to testing snapshot manager when we merge [that PR](https://github.com/insightsengineering/teal/pull/1011). There will be some changes so the tests will have to be rewritten anyway